### PR TITLE
Add pause,resume,priority,count for macOs Swift demo app on Analytics

### PR DESCRIPTION
**Pull request by charmian01**
## Description
Add pause, resume, priority and count features for macOS Swift demo app on Analytics.
It only worked on App Center mode.

### CODE REVIEW

File: SasquatchMac/SasquatchMac/ViewControllers/AnalyticsViewController.swift [hunk 1]
```diff
@@ -25,7 +53,8 @@ class AnalyticsViewController : NSViewController, NSTableViewDataSource, NSTable
     table?.dataSource = self
     NotificationCenter.default.addObserver(self, selector: #selector(self.editingDidBegin), name: .NSControlTextDidBeginEditing, object: nil)
     NotificationCenter.default.addObserver(self, selector: #selector(self.editingDidEnd), name: .NSControlTextDidEndEditing, object: nil)
-  }
+    self.countLabel.stringValue = "Count: \(Int(countSlider.intValue))"
+    }
```
Comment (MatkovIvan) [inline SasquatchMac/SasquatchMac/ViewControllers/AnalyticsViewController.swift]: Format

File: SasquatchMac/SasquatchMac/ViewControllers/AnalyticsViewController.swift [hunk 2]
```diff
@@ -37,7 +66,39 @@ class AnalyticsViewController : NSViewController, NSTableViewDataSource, NSTable
...
 
   @IBAction func trackEvent(_ : AnyObject) {
-    appCenter.trackEvent(name.stringValue, withProperties: properties)
+    let eventName = name.stringValue
+    for _ in 0..<Int(countSlider.intValue) {
+            if priority != .Default {
+                appCenter.trackEvent(eventName, withProperties: properties, flags: priority.flags)
+            } else {
+                appCenter.trackEvent(eventName, withProperties: properties)
+            }
```
Comment (MatkovIvan) [inline SasquatchMac/SasquatchMac/ViewControllers/AnalyticsViewController.swift]: Format

File: SasquatchMac/SasquatchMac/ViewControllers/AnalyticsViewController.swift (hunk 1 above)
Comment (charmian01) [reply SasquatchMac/SasquatchMac/ViewControllers/AnalyticsViewController.swift]: Pushed the new commit to format code, please check.

File: SasquatchMac/SasquatchMac/Base.lproj/SasquatchMac.storyboard [hunk 3]
```diff
@@ -1067,8 +1061,54 @@
...
+                                </constraints>
+                                <textFieldCell key="cell" scrollable="YES" lineBreakMode="clipping" sendsActionOnEndEditing="YES" title="Priority" id="o4O-wp-M4x">
+                                    <font key="font" metaFont="system"/>
+                                    <color key="textColor" name="labelColor" catalog="System" colorSpace="catalog"/>
+                                    <color key="backgroundColor" name="controlColor" catalog="System" colorSpace="catalog"/>
+                                </textFieldCell>
+                            </textField>
+                            <comboBox verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="wg1-7a-ZSe">
+                                <rect key="frame" x="110" y="263" width="311" height="26"/>
+                                <comboBoxCell key="cell" scrollable="YES" lineBreakMode="clipping" selectable="YES" editable="YES" sendsActionOnEndEditing="YES" borderStyle="bezel" title="Default" drawsBackground="YES" completes="NO" numberOfVisibleItems="4" id="Pqq-Uj-PKG">
```
Comment (MatkovIvan) [inline SasquatchMac/SasquatchMac/Base.lproj/SasquatchMac.storyboard]: It should not be editable

File: SasquatchMac/SasquatchMac/ViewControllers/AnalyticsViewController.swift [hunk 4]
```diff
@@ -8,15 +8,43 @@ class AnalyticsViewController : NSViewController, NSTableViewDataSource, NSTable
     static let valueCellId = "valueCellId"
   }
 
+    enum Priority: String {
```
Comment (jaelim-ms) [inline SasquatchMac/SasquatchMac/ViewControllers/AnalyticsViewController.swift]: This block isn't formatted properly. Please run `clang-format-changed-files.sh` script in the root folder of the repo.

File: SasquatchMac/SasquatchMac/ViewControllers/AnalyticsViewController.swift [hunk 5]
```diff
@@ -37,7 +66,40 @@ class AnalyticsViewController : NSViewController, NSTableViewDataSource, NSTable
...
+  }
+
+  @IBAction func countChanged(_ sender: Any) {
+    self.countLabel.stringValue = "Count: \(Int(countSlider.intValue))"
+  }
+
+  @IBAction func priorityChanged(_ sender: NSComboBox) {
+    switch(self.priorityValue.stringValue)
+    {
+    case "Normal":
```
Comment (jwallra) [inline SasquatchMac/SasquatchMac/ViewControllers/AnalyticsViewController.swift]: Can't we use the enum for the case values too?  e.g. `case Priority.Normal:`

File: SasquatchMac/SasquatchMac/ViewControllers/AnalyticsViewController.swift (hunk 4 above)
Comment (jaelim-ms) [reply SasquatchMac/SasquatchMac/ViewControllers/AnalyticsViewController.swift]: Sorry for confusion. The script only works for Objective C files.

File: SasquatchMac/SasquatchMac/Base.lproj/SasquatchMac.storyboard (hunk 3 above)
Comment (charmian01) [reply SasquatchMac/SasquatchMac/Base.lproj/SasquatchMac.storyboard]: Changed `editable `value to "NO"  for `comboBoxCell`

File: SasquatchMac/SasquatchMac/ViewControllers/AnalyticsViewController.swift (hunk 5 above)
Comment (charmian01) [reply SasquatchMac/SasquatchMac/ViewControllers/AnalyticsViewController.swift]: Ok, but it need a "String" type , so I used `case Priority.Normal.rawValue` now.

File: SasquatchMac/SasquatchMac/ViewControllers/AnalyticsViewController.swift (hunk 4 above)
Comment (charmian01) [reply SasquatchMac/SasquatchMac/ViewControllers/AnalyticsViewController.swift]: I fixed indentations for that, please check.

**Pull request merged**